mail-client/mutt/mutt-1.6.1.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="Small but very powerful text-based mail client"
+DESCRIPTION="A text-based email client for Unix-like systems. \
+It was originally written by Michael Elkins in 1995 and released \
+under the GNU General Public License version 2 or any later \
+version.
+
+The Mutt slogan is All mail clients suck. This one just sucks less."
+HOMEPAGE="http://www.mutt.org"
+COPYRIGHT="1996-2014 Michael R. Elkins
+1996-2002 Brandon Long
+1997-2009 Thomas Roessler
+1998-2005 Werner Koch
+1999-2014 Brendan Cully
+1999-2002 Tommi Komulainen
+2000-2004 Edmund Grimley Evans
+2006-2009 Rocco Rutte
+2014-2015 Kevin J. McCarthy
+"
```

Comment:
Quote goes on previous line. And all the continuation COPYRIGHT lines should be indented by 1 tab.